assets/js/pages/topic.js

```diff
@@ -22,7 +22,7 @@ window.bootTopic = function () {
   };
 
 
-  if ($("#postExampleText").length >= 1) {
+  if ($("#postExampleText").length >= 1 && !(urlParam("viewer_pane")==1)) {
```

Comment:
I could post examples with it when I tried, the code in the if-statement is just for markdown in Datacamp Light no?